docker-compose.yml

```diff
@@ -1,10 +1,12 @@
-version: '3.1'
+version: '2'
```

Comment:
What's the problem with version 3? Don't we want to work with the latest? 

I'll note that 3.1 works with docker engine version 1.13.1+